tests/testsuite/check.rs

```diff
@@ -64,7 +64,7 @@ fn check_fail() {
 
     foo.cargo("check")
         .with_status(101)
-        .with_stderr_contains("[..]this function takes 0 parameters but 1 parameter was supplied")
+        .with_stderr_contains("[..]this function takes 0")
```

Comment:
```suggestion
        .with_stderr_contains("[..]this function takes 0[..]")
```

It has to match an entire line.

Reply:
done